lib/overcommit/subprocess.rb

```diff
@@ -32,8 +32,12 @@ def spawn(args)
       # Spawns a new process in the background using the given array of
       # arguments (the first element is the command).
       def spawn_detached(args)
-        # Dissociate process from parent's input/output streams
-        Process.detach(Process.spawn({}, *args, [:in, :out, :err] => '/dev/null'))
+        process = ChildProcess.build(*args)
```

Comment:
Never mind, I ended up updating it myself in 3c15058 since I was working in the code anyway. Thanks!